### DISCUSSION TITLE
Application > Slideover > Chat - Spelling and question about scrolling...

### DISCUSSION THREAD

**Discussion by tvl83** [primefaces/community#813] [PrimeReact Blocks] [answered]
Minor thing just spelling.

_Messages are end-to-end encrypted. No one **expect** you and receiver can read them. None of messages are saved to our servers._

that is at the top of the chat window when clicking on other users except Annette Black. I believe the word that should be there is `except`.

---

Also the last two blocks on the application/slideover page are named `Tabs`. The last one is likely "Forms" or something related as it has a form on the slideover.

---

My other question is about making this a scrolling area for the chats. I implemented this with some modifications I needed but I iterate over some items and create several chat boxes. It scrolls the entire page instead of only the chat window. Can this be updated to have more chats than the height of the window and add a scroll bar ?

---

Thank you. I love the prime blocks they have been very useful.

**Comment by mertsincan** [comment]
Welcome! Thanks a lot for your interest!
@nitrogenous @gucal Can you assist with this ticket?

**Comment by gucal** [comment, marked as answer]
Hi there,
First of all, thank you for your feedback and suggestions for changes. We will implement the changes you suggestion. 

You can try this solution for your question:

```
  <div id="slideover-9" className="hidden bg-surface-0 dark:bg-surface-800 h-full absolute top-0 right-0 shadow w-[30rem] overflow-auto">
```

You can add `overflow-auto` class to the div with slideover-9 id. It will work as you expect. 

Thank you, best regards!
